package.json

```
@@ -5,17 +5,24 @@
   "main": "rubiconmd.js",
   "dependencies": {},
   "devDependencies": {
+    "@babel/cli": "^7.4.3",
+    "@babel/core": "^7.4.3",
+    "@babel/polyfill": "^7.4.3",
+    "@babel/preset-env": "^7.4.3",
     "del": "^2.2.0",
     "gulp": "^3.9.1",
     "gulp-concat": "^2.6.0",
     "gulp-jsdoc3": "^0.2.1",
     "gulp-uglify": "^1.5.3",
     "jsdoc": "^3.4.0",
+    "uglify-js": "^3.5.3",
     "yargs": "^4.6.0"
   },
   "scripts": {
     "test": "non",
-    "build": "gulp build --env='prod'"
+    "build": "gulp build --env='prod'",
+    "build-sentry": "./node_modules/.bin/babel app/dependencies/sentry.js -o app/dependencies/sentry-compiled.js",
```

You don't need to specify the binary path on `package.json`, the following should work:
```suggestion
    "build-sentry": "babel app/dependencies/sentry.js -o app/dependencies/sentry-compiled.js",
```

Unless I'm mistaken, in that case it's still your fault